examples/prologue_france/prologue.catala_fr

```diff
@@ -9,4 +9,24 @@ déclaration énumération Collectivité :
   -- Métropole
   -- SaintPierreEtMiquelon
   -- Mayotte
+
+déclaration champ d'application CalculAgeInfEq:
+  entrée date_naissance contenu date
+  entrée date_courante contenu date
+  entrée années contenu durée
+  résultat r contenu booléen
+
+champ d'application CalculAgeInfEq:
+  définition r égal à date_naissance + années <= date_courante
+  date arrondi croissant
+
+déclaration champ d'application CalculAgeSupStrict:
+  entrée date_naissance contenu date
+  entrée date_courante contenu date
+  entrée années contenu durée
+  résultat r contenu booléen
+
+champ d'application CalculAgeSupStrict:
+  définition r égal à date_naissance + années > date_courante
+  date arrondi croissant
```

Comment:
Why did you create a separate scope to do this ? The code for the date checking is in `prologue.catala_fr` so not justified by any legal rule. We had said that we wanted `date rounding increasing` inside `scope` blocks and not `declaration scope` blocks in order to justify them by legal rules. 

In this example I would not have created these two extra scope and just would have added `date arrondi croissant` to the scope `ÉligibilitéAidesPersonnelleLogement`.

Reply:
Here is my rationale for this approach: I don't want all date computations within `ÉligibilitéAidesPersonnelleLogement` to use the same rounding mode. For age computations I'm pretty sure rounding up is the correct way to do it (although I don't know the exact law defining this, hence the prologue definition), but what about other cases?

Reply:
Ok I understand better. We'll ask them officials the question about date rounding to know more :)

Reply:
@AltGr there is currently no way to do it. I think it should at least be added when functions of multiple arguments can be defined and called in toplevel definitions, what do you think?

Reply:
Regarding documentation, where is it best to explain these behaviors?

Reply:
Sorry, I missed the support for that. I opened #431.